### ISSUE TITLE
OdooSession is not longer json serialisable. 

### ISSUE THREAD

**Issue by adar2378** [ERP-Ukraine/odoo-rpc-dart#63]
Hi, I am using hydrated_bloc to store OdooSession. However, I cannot do it using odooSession because I am receiving this error in the newer versions of odoo_rpc package. The company record is the problem as it is not JSON serializable.  

Error message:
```
Converting object to an encodable object failed: Record (id: 1, name: Company Name)
```

**Comment by lem8r** [comment]
Hi, the structure of OdooSession has changed. I have no idea why there is an error regarding company name. Maybe you need to debug get.

See recent changes https://github.com/ERP-Ukraine/odoo-rpc-dart/compare/10226e2afafa5e7db3ca1061c1897d681af833d6..master

**Comment by lem8r** [comment]
Hi,

This issue has been fixed in 0.5.8
